### PR TITLE
Apply checkstyle to all modules

### PR DESCRIPTION
**Pull request by liran-funaro**
Fixes issue #153 

- Apply checkstyle to all modules
- Add "WhitespaceAround" to checkstyle rules

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by sanastas** [comment]
Looks like there were check-style errors :(

**Comment by liran-funaro** [comment]
> Looks like there were check-style errors :(

Yeah. I didn't fix the issues yet. Now there are no errors.

File: core/src/main/java/com/yahoo/oak/Chunk.java [hunk 1]
```diff
@@ -762,7 +759,7 @@ DescendingIter descendingIter(ThreadContext ctx, K from, boolean fromInclusive,
     // specifier whether the end boundary check needs to be performed on the current scan output
     enum IterEndBoundCheck {
         NEVER_END_BOUNDRY_CHECK,
```
Comment (sanastas) [inline core/src/main/java/com/yahoo/oak/Chunk.java:761]: Maybe also here BOUNDARY instead of BOUNDRY?

**Review by sanastas — APPROVED** [review]
Very good contribution, minor comment below.

File: core/src/main/java/com/yahoo/oak/Chunk.java (hunk 1 above)
Comment (liran-funaro) [reply core/src/main/java/com/yahoo/oak/Chunk.java]: fixed

**Pull request merged**